src/guide/scaling-up/testing.md

```diff
@@ -49,30 +92,75 @@ Vitest is relatively new and is still undergoing rapid development. While it is
 
 ## Component Testing
 
-In Vue applications, components are the main building blocks of the UI. Components are therefore the natural unit of isolation when it comes to visual and interaction tests. From a granularity perspective, component testing sits somewhere above unit testing and can be considered a form of integration testing. In some cases we will be testing a single component, but in other cases we could be testing a tree of components to make sure they are integrated correctly.
+In Vue applications, components are the main building blocks of the UI. Components are therefore the natural unit of isolation when it comes to validating your application's behavior. From a granularity perspective, component testing sits somewhere above unit testing and can be considered a form of integration testing. Much of your Vue Application should be covered by a component test and we recommend that each Vue component has its own spec file.
+
+Component tests should catch issues relating to your component's props, events, slots that it provides, styles, classes, lifecycle hooks, and more.
 
-Component tests should focus on the component's public interfaces rather than internal implementation details. Or, in other words, **test what a component does, not how it does it**.
+They should not mock child components, but instead test the interactions between your component and its children by interacting with the components as a user would. For example, a component test should click on an element like a user would instead of programmatically interacting with the component.
```

Comment:
This line feels very confusing for me. Description of **Unit**-tests never explicitly mentions that components could also be tested in this way (by using shallow rendering). So basically `shallow` style of testing is silently ignored

Additionally this requirement feels extremely **strict**  - considering this guide will be an intro to Vue testing for many people, it would be great to be both actionable and not biased towards such things

Reply:
Oh, we can add copy about `shallowMount` into Unit Tests if we'd like. I didn't mention it, but not on purpose.